ush/parsing_namelists_FV3.sh

```diff
@@ -434,7 +428,7 @@ if [[ ${DO_CA:-"NO"} = "YES" ]]; then
 EOF
 fi
 
-if [[ ${DO_LAND_PERT:-"NO"} = "YES" ]]; then
+if [[ "${DO_LAND_PERT:-}" == "YES" ]]; then
```

Comment:
This change should be undone. If `DO_LAND_PERT` is undefined, this will have the correct logic but print a syntax error in the log about `==` not being a unary operator because there is nothing on the left side of the operator.

Reply:
done.